cime/config/e3sm/testmods_dirs/allactive/hommexx/shell_commands

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
Michael introduced this first and he had reasons, but my understanding is that we don't need this code. If so, can we not introduce it in cime?

Reply:
I'm not an expert of cime. Not really understanding this, I left it there. If you can confirm is not needed, then sure, let's remove it. @jgfouca any comment on this?

Reply:
I don't think a testmod is the correct way to implement this. This should be in the build system.

Reply:
For the preqx_acc target, the method is to create your case normally and issue the xlmchange command to change CAM_TARGET by hand.   We don't have syntax to set "CAM_TARGET" at the create_newcase stage.